gtsfm/frontend/matcher/dummy_matcher.py

```diff
@@ -4,6 +4,7 @@
 """
 import numpy as np
```

Comment:
Do we still need the dummy matcher class, now that we have a real one? can we delete it?

Reply:
It helps to define generic unit tests.

Reply:
cant a generic unit test work on our actual matcher implementation?

Reply:
How about moving the dummy class to the unit test file? It makes us lean atleast in the number of files. I have pushed this change for now